pygbm/histogram.py

```diff
@@ -20,6 +20,40 @@ def _build_ghc_histogram_naive(n_bins, sample_indices, binned_feature,
     return histogram
 
 
+@njit
+def _build_ghc_histogram_unrolled_fast(n_bins, parent_histogram,
+                                       sibling_histogram):
+    histogram = np.zeros(n_bins, dtype=HISTOGRAM_DTYPE)
+    unrolled_upper = (n_bins // 4) * 4
+
+    for i in range(0, unrolled_upper, 4):
```

Comment:
Are you sure that unrolling bring any speed up for just a short iteration? n_bins is expected to be 255 max.

Reply:
No idea... I just thought it coudn't hurt but indeed I don't think it's really useful.